scipy/optimize/setup.py

```diff
@@ -93,6 +93,20 @@ def configuration(parent_package='', top_path=None):
         join('slsqp', x) for x in sources], **numpy_nodepr_api)
     ext._pre_build_hook = gfortran_legacy_flag_hook
 
+    sources = ['direct_wrap.c', 'DIRect.c',
+               'DIRsubrout.c', 'DIRserial.c']
+    headers = ['direct.h', 'direct-internal.h']
+    config.add_library('_direct_lib',
+                       sources=[join('_direct', x) for x in sources],
+                       headers=[join('_direct', x) for x in headers],
```

Comment:
The failure to find `Python.h` in CI looks like it's due to a missing path in `add_library`. See if something like https://github.com/rgommers/scipy/commit/92a47a38e67f13ad72151e046fbc2c72c336edfe fixes it.

Reply:
Seems to be working. Thanks.